examples/pipeline_tacotron2/README.md

```diff
@@ -141,3 +141,95 @@ python train.py \
     --checkpoint-path ./english_phonemes_wavernn_ckpt.pth \
     --dataset-path ./
 ```
+
+
+## Text-to-speech pipeline
+
+Here we present an example of how to use Tacotron2 to generate audio from text.
+For the text-to-speech pipeline, with an input text, 
+Tacotron2 will first generate the corresponding mel spectrogram.
```

Comment:
nit: for those who do not know how TTS works, giving a brief description of process (text preprocessing, feature extraction, spectrogram generation, time-domain conversion) would be nice. (if not done so)
Maybe it is better suited for tutorial.

Reply:
I've updated the README [here](https://github.com/pytorch/audio/pull/1654/files#diff-a17fd0fdb11c8336af63efc614215c0ef98e67d810b77c5c5e8d87a012ca64f1R148-R157) and hopefully its a bit clearer.

And yes, when the tutorial is ready we should definitely add the link here.